substrate/primitives/weights/src/lib.rs

```diff
@@ -18,8 +18,7 @@
 //! # Primitives for transaction weighting.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-// TODO remove once `OldWeight` is gone. I dont know why this is needed, maybe by one of the macros
-// of `OldWeight`.
+// TODO remove once weight_meter::WeightMeter::check_accrue is removed
```

Comment:
Maybe you can also delete that?  
It says that it would be removed after December 2023.

Reply:
got it, let me try removing it